Integrations code sanitization

RamosFe opened the pull request:
|Related issue|
|---|
|https://github.com/wazuh/wazuh/issues/20758|


## Description

Code sanitization for the `/integrations` folder.

nico-stefani commented on integrations/tests/test_maltiverse.py:
Please add tests for the new functions of maltiverse.

nico-stefani commented on integrations/tests/test_virustotal.py:
Add tests for the new functions or split the existing ones here.

nico-stefani commented on .ruff.toml:
Remove this file for now. We will include it when we close the epic issue.

RamosFe commented:
## Update

Added tests to `maltiverse`:
https://github.com/wazuh/wazuh/blob/3009a61fa3e061c90c3e5957f2570fccc498a5a0/integrations/tests/test_maltiverse.py#L254-L342

Refactored already existing tests for `virustotal`:
https://github.com/wazuh/wazuh/blob/3009a61fa3e061c90c3e5957f2570fccc498a5a0/integrations/tests/test_virustotal.py#L274-L306

## Results

```
(venv) federamos@pop-os:~/Documents/Wazuh/Repositories/wazuh/integrations$ python3 -m pytest tests/
========================================================================================================== test session starts ==========================================================================================================
platform linux -- Python 3.9.18, pytest-5.4.3, py-1.10.0, pluggy-0.13.1
rootdir: /home/federamos/Documents/Wazuh/Repositories/wazuh/integrations, inifile: pytest.ini
plugins: metadata-2.0.4, html-2.1.1, tavern-1.2.2, asyncio-0.15.1, cov-2.12.0
collected 122 items                                                                                                                                                                                                                     

tests/test_maltiverse.py ..............................................................                                                                                                                                           [ 50%]
tests/test_pagerduty.py ..........                                                                                                                                                                                                [ 59%]
tests/test_shuffle.py ..................                                                                                                                                                                                          [ 73%]
tests/test_slack.py ..........                                                                                                                                                                                                    [ 81%]
tests/test_virustotal.py ......................                                                                                                                                                                                   [100%]

========================================================================================================== 122 passed in 3.72s ==========================================================================================================
```

EduLeon12 reviewed (dismissed):
LGTM !

Selutario commented on integrations/shuffle.py:
```diff
@@ -247,36 +262,38 @@ def get_json_alert(file_location: str) -> any:
+
+    Returns
+    -------
+    {}: any
```
```suggestion
    dict: any
```

Selutario reviewed (changes requested):
GJ. However, after reviewing how this new format actually looks, I think we should add this rule to the `.ruff.toml` file (as Nico mentioned, do not include the file here yet): 
- [multi-line-summary-first-line (D212)](https://docs.astral.sh/ruff/rules/multi-line-summary-first-line/)

We are following the numpy docstring format and it always contain a short summary on the first line as shown [here](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard):
![image](https://github.com/wazuh/wazuh/assets/23361101/2124f596-72a3-48e4-be6e-453bd3a69699)

Update these files to comply with this additional requirement please.

Selutario commented on integrations/maltiverse.py:
```diff
@@ -5,8 +5,7 @@
 
-"""
-What is Maltiverse?
+"""What is Maltiverse?
```
This module docstring looks slightly strange when applying D212 rule since the first line in this case is not a summary but a title.

Selutario commented:
Merging since the failing checks are unrelated to these changes.
